communications/rfc.rst

```diff
@@ -16,13 +16,32 @@ On this page:
 - :ref:`rfc-jira`.
 - :ref:`rfc-cases`.
 - :ref:`rfc-discussion`.
-- :ref:`rfc-exceptions`.
 - :ref:`rfc-formalities`.
 
 .. seealso::
 
    :doc:`rfd` has a similar name, but has to do with convening discussions across DM teams.
 
+.. _rfc-cases:
+
+Standard procedures that require an RFC
+=======================================
+
+The following procedures **always** require an RFC:
+
+- :doc:`/stack/adding-a-new-package`.
```

Comment:
Anywhere? Or solely science pipelines?

Reply:
That page says anywhere. I think we could discuss changing that, but not on this PR.